app/models/book.rb

```diff
@@ -7,4 +7,21 @@ class Book < ApplicationRecord
   has_many :book_authors
   has_many :reviews
   has_many :authors, through: :book_authors
+
+  def self.by_page_count_ascending
```

Comment:
for that matter, you could DRY up these four methods by passing which field you want to sort by, plus the direction of the sort.